### PR TITLE
FIX: Use `testing` not `sample`, dont require fsaverage

### PR DESCRIPTION
**Pull request by larsoner**
Fixes a regression introduced in #4894 where we used `sample.data_path()`, causing too much disk space usage (causing failures like this: https://travis-ci.org/mne-tools/mne-python/jobs/357552658). Also changed to fake `fs_home` as `testing.data_path()` so FreeSurfer is no longer required.

@christianbrodbeck can you take a quick look when you get a chance?

### CODE REVIEW

**Comment by larsoner** [comment]
@christianbrodbeck I don't see any comment, maybe it got lost?

File: mne/coreg.py [hunk 1]
```diff
@@ -1048,56 +1060,58 @@ def scale_mri(subject_from, subject_to, scale, overwrite=False,
...
 
-    # BEM files [in m]
+    logger.debug('BEM files [in m]')
     for bem_name in paths['bem']:
-        scale_bem(subject_to, bem_name, subject_from, scale, subjects_dir)
+        scale_bem(subject_to, bem_name, subject_from, scale, subjects_dir,
+                  verbose=False)
 
-    # fiducials [in m]
+    logger.debug('fiducials [in m]')
```
Comment (christianbrodbeck) [inline mne/coreg.py]: Yeah the comment probably got lost, I just said that the `[in m]` are more code-internal notes and probably not relevant to log messages.

Comment (larsoner) [reply mne/coreg.py]: `logger.debug` are more or less for internal notes :)

Comment (christianbrodbeck) [reply mne/coreg.py]: It's fine with me :)

**Comment by larsoner** [comment]
Merging quickly so that other PRs can be green

**Pull request merged**